data/json/effects.json

```diff
@@ -1746,6 +1758,12 @@
         "name": [ "Unhappy", "Unhappy", "Unhappy", "Unhappy", "Unhappy",
                   "Sad", "Sad", "Sad", "Sad", "Sad",
                   "Depressed" ],
+        "desc": [ "You are not content with your life.", "You are not content with your life.",
+                  "You are not content with your life.", "You are not content with your life.",
+                  "You are not content with your life.", "This is not what you planned for your life.",
+                  "This is not what you planned for your life.", "This is not what you planned for your life.",
+                  "This is not what you planned for your life.", "This is not what you planned for your life.",
+                  "When will this end?." ],
```

Comment:
Recommend `When will all your suffering end?`

Reply:
Saying that in third person, while correct in our books, feels a bit disconnected from the character. I used this way of pronouncing it to make it more personal, emotional. But I will go by your suggestion.